src/interface/events.ts

```diff
@@ -20,8 +20,10 @@ export enum InterfaceEventName {
   TOKEN_IMPORTED = 'Token Imported',
   TOKEN_SELECTED = 'Token Selected',
   TOKEN_SELECTOR_OPENED = 'Token Selector Opened',
-  UNISWAP_WALLET_APP_DOWNLOAD_OPENED = 'Uniswap Wallet App Download Opened',
+  UNISWAP_ANDROID_APP_DOWNLOAD_CLICKED = 'Uniswap Android App Download Link Clicked',
+  UNISWAP_IOS_APP_DOWNLOAD_CLICKED = 'Uniswap iOS App Download Link Clicked',
   UNISWAP_WALLET_MICROSITE_OPENED = 'Uniswap Wallet Microsite Opened',
+  UNIWALLET_CONNECT_MODAL_OPENED = 'Uniswap wallet modal opened',
```

Comment:
sentence-case string because there is an existing event with this name hardcoded in interface -- want to avoid breaking dashboards

Reply:
Where was this event defined previously?

Reply:
https://github.com/Uniswap/interface/blob/main/src/components/AccountDrawer/UniwalletModal.tsx#L60